static/app/views/traces/content.tsx

```diff
@@ -348,9 +389,13 @@ function TraceRow({defaultExpanded, trace}: {defaultExpanded; trace: TraceResult
       </StyledPanelItem>
       <StyledPanelItem align="left" overflow>
         <Description>
-          {trace.project ? (
-            <ProjectRenderer projectSlug={trace.project} hideName />
-          ) : null}
+          <ProjectBadgeWrapper>
+            {traceProjects.length > 0 ? (
+              <ProjectsRenderer projectSlugs={traceProjects} />
+            ) : trace.project ? (
+              <ProjectsRenderer projectSlugs={[trace.project]} />
+            ) : null}
```

Comment:
Maybe just arrayify it inline? nit though

```suggestion
            <ProjectsRenderer projectSlugs={traceProjects.length ? traceProjects : [trace.project]} />
```